### DISCUSSION TITLE
Multiple filesystems

### DISCUSSION THREAD

**Discussion by remram44** [seaweedfs/seaweedfs#2995] [Q&A] [answered]
It seems that I can run multiple Filer processes with a single set of masters and volume servers. However the Filers will discover each other via the master and synchronize, if I understand [Filer Store Replication](https://github.com/chrislusf/seaweedfs/wiki/Filer-Store-Replication) correctly.

Is there a way to have multiple separate filesystems backed by the same volume servers? For example by running multiple Filers, but avoiding this synchronization mechanism? This is generally desirable because the separate Filers are independent giving better performance than going through a single process.

For instance [support for multiple filesystems](https://docs.ceph.com/en/pacific/cephfs/multifs/) was added to Ceph as of version 16 "pacific".

**Comment by chrislusf** [comment, marked as answer]
there is a plan to add namespaces for each group of filers.

**Comment by chrislusf** [comment]
Added "filerGroup" option when starting filers, available in 3.01
